performance/sqlserver/memory_settings.rst

```diff
@@ -0,0 +1,21 @@
+Memory Settings
+===============
+
+The total ammoun of memory needed by SQL Server depends on your use case and (expected) workload. SQL Server will claim memory from the OS when needed but will not give it back to the OS once claimed. When not specifying a limit you will run the risk of starving the host server and OS of enough free memory to operate efficently. It is therefore strongly advised to limit the maximum ammount SQL Server will use.
+
+To set a fixed amount of memory using SSMS [#]_:
+
+#. In Object Explorer, right-click a server and select Properties.
+#. Click the Memory node.
+#. Under Server Memory Options, enter the amount that you want for Minimum server memory and Maximum server memory.
```

Comment:
- Capitalization: 'Minimum' -> 'minimum'
- Capitalization: 'Maximum' -> 'maximum'

Reply:
It references the field names in the screenshot. I have put quotes around in it and fully matched the complete name which hopefully makes it a bit more clearer.